website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -1,30 +1,39 @@
 ---
 title: Shared Signals Framework (SSF) Provider
 sidebar_label: SSF Provider
+tags: [ssf, backchannel, provider, "security event tokens"]
 authentik_version: "2025.2.0"
 authentik_enterprise: true
 authentik_preview: true
 ---
 
-Shared Signals Framework (SSF) is a common standard for sharing asynchronous real-time security signals and events across multiple applications and an identity provider. The framework is a collection of standards and communication processes, documented in a [specification](https://openid.net/specs/openid-sharedsignals-framework-1_0-ID3.html). SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
+Shared Signals Framework (<abbr>SSF</abbr>) is a collection of standards for sharing asynchronous security signals and events across multiple applications and an identity provider. SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
 
-## About Shared Signals Framework
+## What are security signals?
```

Comment:
While that title is good, I want to leaveit as is, with the word "About" as a cognitive hint that this is the conceptual section... it's in the template for same reason. (Though yes, the proposed title also is clearly conceptual. I'd just like to stay consistent with our headers/titles.)